From 8e3071660ef78f8da46225b51e811b30006f29f2 Mon Sep 17 00:00:00 2001
From: Chris Wilson <chris@chris-wilson.co.uk>
Date: Sun, 19 Sep 2010 14:46:27 +0100
Patch-mainline: 2.6.37
References: fate#310916
Git-commit: e070868ef2101ce548d4fbb25edfd301e59fb719
Subject: [PATCH 1438/2588] drm/i915/ringbuffer: Mark the initialisation
 structs as constant.

Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
(cherry picked from commit e070868ef2101ce548d4fbb25edfd301e59fb719)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/i915/intel_ringbuffer.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_ringbuffer.c b/drivers/gpu/drm/i915/intel_ringbuffer.c
index 478406d..f0b44d3 100644
--- a/drivers/gpu/drm/i915/intel_ringbuffer.c
+++ b/drivers/gpu/drm/i915/intel_ringbuffer.c
@@ -792,7 +792,7 @@ void intel_fill_struct(struct drm_device *dev,
 	intel_ring_advance(dev, ring);
 }
 
-static struct intel_ring_buffer render_ring = {
+static const struct intel_ring_buffer render_ring = {
 	.name			= "render ring",
 	.id			= RING_RENDER,
 	.regs                   = {
@@ -830,7 +830,7 @@ static struct intel_ring_buffer render_ring = {
 
 /* ring buffer for bit-stream decoder */
 
-static struct intel_ring_buffer bsd_ring = {
+static const struct intel_ring_buffer bsd_ring = {
 	.name                   = "bsd ring",
 	.id			= RING_BSD,
 	.regs			= {
@@ -948,7 +948,7 @@ gen6_bsd_ring_dispatch_gem_execbuffer(struct drm_device *dev,
 }
 
 /* ring buffer for Video Codec for Gen6+ */
-static struct intel_ring_buffer gen6_bsd_ring = {
+static const struct intel_ring_buffer gen6_bsd_ring = {
        .name			= "gen6 bsd ring",
        .id			= RING_BSD,
        .regs			= {
-- 
1.7.6

